libweasyl/libweasyl/staff.py

```diff
@@ -1,70 +1,47 @@
 """
-Dictionaries of Weasyl staff.
+Global variables of staff.
 
-This module must be initialized with a call to `staff._init_staff()`
-with the path to a yaml config file where staff members are identified
-by their user id. Example contents of such a config file might look as
-follows:
+This module must be initialized by the following call:
 
-    directors:
-        - 1014   # Fiz
-        - 2061   # Ikani
+_init_staff(directors=staff_config_dict['directors'],
+            technical_staff=staff_config_dict['technical_staff'],
+            admins=staff_config_dict['admins'],
+            mods=staff_config_dict['mods'],
+            developers=staff_config_dict['developers'])
 
-    technical_staff:
-        - 5756   # Weykent
-
-    admins:  # Directors and technical_staff also have admin privs.
-        - 23613  # Hendikins
-        - 3      # Kihari
-
-    mods:  # Admins also have mod privs.
-        - 40212  # pinardilla
-
-    developers:
-        - 38623  # 8BitFur
-        - 34165  # Charmander
-        - 15224  # Foximile
-        - 2475   # Kailys
-        - 8627   # Kauko
-
-We recommend storing staff members in lexicographic order by name
-within each group.
+The staff_config_dict is generated by weasyl/read_staff_yaml.py; see that file
+for further information on the formatting of the dict.
 """
 
-import yaml
-
-
-DIRECTORS = set()
-""" Directors have the same powers as admins. """
-
-
-TECHNICAL = set()
-""" Technical staff can moderate all content and manage all users. """
-
-
-ADMINS = set()
-""" Site administrators can update site news and moderate user content. """
 
+def _init_staff(**kwargs):
```

Comment:
Any reason not to have `(directors, technical_staff, admins, mods, developers)`?